aws/templates/id/id_contenthub.ftl

```diff
@@ -56,7 +56,7 @@
             },
             "Attributes" : {
                 "ENGINE" : getExistingReference(id, ENGINE_ATTRIBUTE_TYPE),
-                "URL"    : getExistingReference(id, URL_ATTRIBUTE_TYPE),
+                "REPOSITORY" : getExistingReference(id, REPOSITORY_ATTRIBUTE_TYPE),
                 "BRANCH" : getExistingReference(id, BRANCH_ATTRIBUTE_TYPE),
                 "PREFIX" : getExistingReference(id, PREFIX_ATTRIBUTE_TYPE)
```

Comment:
How does prefix differ from path?

Reply:
Prefix was for the Hub side and could be a folder or appended to a name. Path is for the node side to identify who the node is. Since they are used together in the scripts I wanted to seperate them.